dagmc_stats/generate_stats.py

```diff
@@ -190,6 +198,11 @@ def collect_statistics(my_core, root_set, tar_meshset, display_options):
         data[c_key] = dagmc_stats.get_coarseness(my_core, root_set, 
                                                                 entityset_ranges['Surfaces'], dagmc_tags['geom_dim'])
         stats[c_key] = get_stats(data[c_key])
+    
+    if display_options['R']:
+        r_key = 'R'
```

Comment:
is `r_key` used anywhere else? I think you can just use `data['R']` and `stats['R']`.

Reply:
Since the previous options are added in this way, I added like this to keep it consistent